packages/forms/backend/sirius-components-collaborative-forms/src/main/resources/schema/form.graphqls

```diff
@@ -77,6 +77,11 @@ type ContainerBorderStyle {
   radius: Int!
   size: Int!
   lineStyle: ContainerBorderLineStyle!
+  justifyContent: ContainerContentAlignment!
+  alignItems: ContainerContentAlignment!
```

Comment:
While it may work for now, this may quickly not work anymore since popular options like `space-evenly` or `space-between` are not available on both properties. It would be better to have one enumeration per property because they are not the same.

Reply:
I maded two seperate enum.